src/main/java/TaskList.java

```diff
@@ -0,0 +1,178 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class to handle all the tasks.
+ */
+public class TaskList {
+    private static final String ADD_TODO_COMMAND = "todo";
+    private static final String ADD_DEADLINE_COMMAND = "deadline";
+    private static final String ADD_EVENT_COMMAND = "event";
+
+    List<Task> lst = new ArrayList<>();
+
+    /**
+     * Iterates and returns enumerated list of tasks.
+     */
+    public void iterateList() {
+        System.out.println("Here are the items in your list:");
+        int i = 1;
+        for (Task item : lst) {
+            System.out.println(i + ". " + item.toString());
+            i++;
+        }
+    };
+
+    public List<Task> fetchTasks() {
```

Comment:
Any reason for the choice of name for this method? The name suggests that it will return Tasks, but instead if gets the `lst` and returns it. Would it be more appropriate to call it `getXXX` instead? Just some thoughts!

Reply:
Perhaps it could be named as `displayList`?